src/dataDisplay/IconText/index.tsx

```diff
@@ -1,25 +1,31 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import { ThemeColors, ThemeIconSize, ThemeTextSize } from '../../theme';
+import {
+  ThemeColors,
+  ThemeIconSize,
+  ThemeMargin,
+  ThemeTextSize,
+} from '../../theme';
 import { Icon, IconType } from '../Icon';
 import Text from '../Text';
 
 type Props = {
   iconType: keyof IconType;
   iconSize: ThemeIconSize;
+  margin: ThemeMargin;
   textSize: ThemeTextSize;
   color?: ThemeColors;
   text: string;
   className?: string;
+  iconSide?: 'left' | 'right';
 };
 
-const StyledIconText = styled.div`
+const StyledIconText = styled.div<{ margin: ThemeMargin }>`
   display: flex;
   align-items: center;
-
-  p {
-    margin-left: 6px;
+  svg {
+    margin: 0 ${({ theme, margin }) => theme.margin[margin]};
```

Comment:
This code has a problem, it's applying margin in both, right and left.
I think this should work like so: 
- if `side` is left, add right-margin
- if `side` is right add left-margin.